lib/tagged_content_exporter.rb

```diff
@@ -63,4 +65,23 @@ def calculate_taxon_depth(taxon_hash, depth = 0)
       calculate_taxon_depth(links["parent_taxons"].first, depth + 1)
     end
   end
+
+  def calculate_taxon_path(taxon_hash, paths = [])
```

Comment:
Could we combine this somehow with the `calculate_taxon_depth` method? It looks like it's doing the same recursion so not sure what the performance hit is when we do it twice